src/redux/migrations.ts

```diff
@@ -1416,5 +1412,5 @@ export const migrations = {
       },
     }
   },
-  166: (state: any) => ({ ...state, app: { ...state.app, hideHomeBalances: false } }),
+  165: (state: any) => ({ ...state, app: { ...state.app, hideHomeBalances: false } }),
```

Comment:
We shouldn't touch past migrations. Only fix forward.

Reply:
It's an issue if we use a key for something, then remove the feature without removing the key from the state and then reuse the same key name for a new thing, that could have a different type.

Reply:
Fixed in 5f66a56f259a5a96ed337bd3a0dbfa283744ed3d!